jibri/Dockerfile

```diff
@@ -1,8 +1,10 @@
 ARG JITSI_REPO=jitsi
 FROM ${JITSI_REPO}/base-java
 
-ARG CHROME_RELEASE=latest
-ARG CHROMEDRIVER_MAJOR_RELEASE=latest
+#ARG CHROME_RELEASE=latest
+#ARG CHROMEDRIVER_MAJOR_RELEASE=latest
```

Comment:
Do we still need these comments? If you want to tell the user that he can change the version and why it is forced to be `78` it should be documented in the README or right here in the file.

Reply:
They are not needed... but I think they serve the purpose of raising an eyebrow and git blame would do the rest. At any rate I hope we don't have to keep it for long...